pkg/dyninst/module/config.go

```diff
@@ -70,12 +74,15 @@ func NewConfig(_ *sysconfigtypes.Config) (*Config, error) {
 		return nil, err
 	}
 
+	cfg := pkgconfigsetup.SystemProbe()
+	sysconfig.Adjust(cfg)
 	c := &Config{
 		Config:             *ebpf.NewConfig(),
 		LogUploaderURL:     withPath(traceAgentURL, logUploaderPath),
 		DiagsUploaderURL:   withPath(traceAgentURL, diagsUploaderPath),
-		SymDBUploadEnabled: pkgconfigsetup.SystemProbe().GetBool("dynamic_instrumentation.symdb_upload_enabled"),
+		SymDBUploadEnabled: cfg.GetBool("dynamic_instrumentation.symdb_upload_enabled"),
 		SymDBUploaderURL:   withPath(traceAgentURL, symdbUploaderPath),
+		SymDBCacheDir:      path.Join(cfg.GetString("run_path"), "symdb_uploads"),
```

Comment:
should we check that this `run_path` exists or is nonempty? Maybe the answer is no.

Reply:
We don't necessarily expect the dir to exist (although I will change the k8s charts to mount it, so there it will exist). We create the dir if it doesn't exist, and we deal with its contents -- generally by deleting everything we don't recognize.
No?